### ISSUE TITLE
Open pointer in new tab shows blank page

### ISSUE THREAD

**Issue by mtrezza** [parse-community/parse-dashboard#1835]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and [Parse Dashboard](https://github.com/parse-community/parse-dashboard/releases). 

### Issue Description
Right-click on a pointer cell and selecting "Open in new tab" opens a blank tab.

### Steps to reproduce
1. right-click on a pointer cell
2. select "Open in new tab"

### Actual Outcome
blank tab opens, e.g. with URL
```
https://example.com/apps/example/browser/_User?filters=%5B%7B%22field%22%3A%22objectId%22%2C%22constraint%22%3A%22eq%22%2C%22compareTo%22%3A%2abc123%22%7D%5D
```

### Expected Outcome
tab should show page filtered for pointer object

### Environment

Dashboard
- Parse Dashboard version: `3.2.0`
- Browser (Safari, Chrome, Firefox, Edge, etc.): `Safari`
- Browser version: `Version 15.0 (16612.1.29.41.4, 16612)`

### Logs

n/a

**Comment by sadakchap** [comment]
It looks to me working fine.

https://user-images.githubusercontent.com/44117648/138402153-27be2a14-d073-4627-89f1-7fed16eb4389.mp4

@mtrezza  could you please confirm this issue once more with latest `alpha` branch ?

**Comment by mtrezza** [comment]
Thanks for asking, it seems to have been fixed already, tested with 3.3.0-alpha.9.
Closing.
